src/tests/utils/routerUtils.test.js

```diff
@@ -1,79 +0,0 @@
-import { searchRouteTree, replaceUrlVariables } from 'console/utils/router';
```

Comment:
Why remove this? Do you plan on writing some other tests? Move them elsewhere? Did I miss something?

Reply:
I wrote some new tests. I deleted this file because all the functions in here were deleted.